tests/verbs/util.py

```diff
@@ -65,17 +73,23 @@ def compare_outputs(
     cols = expected.columns if columns is None else columns
     try:
         assert len(actual) == len(expected)
-        assert len(actual.columns) == len(cols)
-        for column in cols:
-            # dtypes can differ since the test data is read from parquet and our workflow runs in memory
-            assert_series_equal(actual[column], expected[column], check_dtype=False)
     except AssertionError:
-        print("Expected:")
-        print(expected.head())
-        print("Actual:")
-        print(actual.head())
+        print("Expected:", cols)
+        print("Actual:", actual.columns)
         raise
 
+    for column in cols:
+        assert column in actual.columns
+        try:
+            # dtypes can differ since the test data is read from parquet and our workflow runs in memory
+            assert_series_equal(actual[column], expected[column], check_dtype=False)
+        except AssertionError:
```

Comment:
Just curious, if you add `-vvv` when you execute the test, doesn't it automatically give you the expected vs actual?

Reply:
Also you can do 

```py
assert condition, "Error message"
```

if you want a custom error message